crates/wiggle/Cargo.toml

```
@@ -17,14 +17,33 @@ wiggle-macro = { path = "macro", version = "0.27.0" }
 tracing = "0.1.15"
 bitflags = "1.2"
 async-trait = "0.1.42"
+wasmtime = { path = "../wasmtime", version = "0.27.0", optional = true }
```

Could we set `default-features` to `false` here, please? Otherwise this results in the `default` feature of `wasmtime` being forcefully enabled (because of cargo resolver's feature unification), and making it impossible to disable it in embeddings that indirectly make use of `wiggle`.

```suggestion
wasmtime = { path = "../wasmtime", version = "0.27.0", optional = true, default-features = false }
```

---

Hmm this is slightly wrong: should also re-enable manually the `async` feature.

---

Thanks, I missed that.